manim/mobject/mobject.py

```diff
@@ -2011,19 +2016,69 @@ def shuffle(self, recursive=False):
         random.shuffle(self.submobjects)
 
     def invert(self, recursive=False):
+        """Inverts the list of :attr:`submobjects`.
+
+        Examples
+        --------
+
+        .. manim:: InvertSumobjectsExample
+            :save_last_frame:
```

Comment:
Was this intended to be a video? It's unclear what this does from the image.

Reply:
Oh yes, thanks, that was a copy paste  mistake